gulp-tasks/build-packages.js

```diff
@@ -26,6 +26,22 @@ const ERROR_NO_NAMSPACE = oneLine`
 `;
 
 /**
+ * This function takes an object like { nested: { foo: bar } } and a string
+ * like `nested.foo` and returns true if that value is defined on the obejct.
+ */
+const testObjectValueExists = (object, nestedPath) => {
+  const pieces = nestedPath.split('.');
+  let currentRoot = object;
+  pieces.forEach((piece) => {
+    if (!currentRoot.piece) {
+      return false;
+    }
+    currentRoot = object[piece];
+  });
+  return true;
```

Comment:
This function is always going to return `true` since the inner `return false` statement will only return `false` from the `forEach()` loop.